misc/crawl_db.js

```diff
@@ -0,0 +1,68 @@
+var Crawler = require('../src/crawler.js').Crawler
+var _ = require('lodash');
+var nconf = require('nconf');
+var Sequelize = require('sequelize');
+var Promise = Sequelize.Promise;
+var modelsFactory = require('../src/models.js');
+
+/* --------------------------------- CONSTS --------------------------------- */
+
+nconf.argv().env();
+
+/* --------------------------------- HELPERS -------------------------------- */
+
+var saveDB = exports.saveDB = function saveDB(crawlJson, entryIp, dbUrl, onDone) {
```

Comment:
`entryIp` could be removed as a parameter since you can get it from `crawlJson.entry`